integration-tests/extensions-reload.test.ts

```diff
@@ -84,8 +84,11 @@ describe('extension reloading', () => {
       await run.expectText('- hello');
 
       // Update the extension, expect the list to update, and mcp servers as well.
-      await run.sendText('/extensions update test-extension');
-      await run.type('\r');
+      await run.sendKeys('/extensions update test-extension');
+      await run.expectText('/extensions update test-extension');
+      await run.sendKeys('\r');
+      await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Resolved flakiness in this test by sending \r twice here. I don't love that. @scidomino not sure why this would now be required when in alternate buffer mode.